site/docs/v1/tech/apis/_user-registration-combined-request-body.adoc

```diff
@@ -109,7 +109,45 @@ The User's preferred timezone. The string must be in an https://www.iana.org/tim
 [quote]
 `America/Denver` or `US/Mountain`
 
-[field]#users.twoFactorDelivery# [type]#[String]# [optional]#Optional# [default]#defaults to `None`#::
+[field]#user.twoFactor.methods``[x]``.authenticator.algorithm# [type]#[TOTPAlgorithm]# [optional]#Optional#::
+The algorithm used by the TOTP authenticator. With the current implementation, this will always be `HmacSHA1`.
```

Comment:
See comments "The algorithm used by the TOTP authenticator. With the current implementation, this will always be `HmacSHA1`." =>  "The algorithm used by the TOTP authenticator. With the current implementation, this must be `HmacSHA1` if provided. Any other value will be ignored."

Since this is a request, correct?